Command/RemoveExpiredSubmissionsCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use Elasticsearch\Client;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use EMS\CoreBundle\Service\Form\Submission\FormSubmissionService;
+
+class RemoveExpiredSubmissionsCommand extends EmsCommand
```

Comment:
Don't think you  should extend from EmsCommand.
EmsCommand will wait for elasticsearch to be  green, but for this command we don't need elasticsearch. 

Just extend from Symfony\Component\Console\Command\Command and implement EMS\CommonBundle\Command\CommandInterface